packages/public/server/src/module/Common/src/Filter/Slugify.php

```diff
@@ -28,34 +28,51 @@ class Slugify implements FilterInterface
 {
     /**
      * @param string $text
-     * @return bool|mixed
+     * @return bool|string
      */
-    protected static function slugify($text)
+    protected static function slugify(string $text)
```

Comment:
Why is this function prototected? I would do one of the followings:

1. Make this function public and call it in AliasManager via `Slugify::slugify(...)` > maybe move the static function to `Common\Utils` where we can collect utility functions
2. Merge this function into the `filter()` function

I prefer (1) as a solution

Reply:
Since we don't have tests for most stuff in-place, I left everything I didn't need to change untouched. In this specific case, `Slugify` implements the `ZendFilterInterface` and this is why this is called via the public method `filter` instead. Don't ask me whether Zend does anything special when something implements the `ZendFilterInterface`.

Reply:
It seems to be used in a couple of places:
![Screen Shot 2020-11-26 at 16 04 26](https://user-images.githubusercontent.com/1508477/100366445-1d52d380-3001-11eb-80f0-91a4f3f23a74.png)


Though If I understand the documentation correctly, it just allows you to use some built-in helpers of Zend but doesn't matter much otherwise. So we could refactor them and don't use `Zend\Filter\FilterInterface` at all.

Reply:
I meant `Slugify` is only used at `SubjectHelper` -> Maybe the other places will help identify why `Zend\Filter\FilterInterface` is needed. I would guess that it can be added to a Form...